### ISSUE TITLE
Stuck on subtitle keyword search

### ISSUE THREAD

**Issue by Kapplahmastah** [Hellowlol/bw_plex#28]
Hey, thanks for the work you have put into this, I really like the idea behind this project, and it seems like a really smart way to handle it. Unfortunately I am unable to make it work properly. I get as far as the line where it's trying to look trough a subtitle file, but at that moment it is stuck until I try to play another episode. It does find the right time in the previous line;
"bw_plex :: DEBUG :: Matching windows are [['03:15', '03:18', '00:02'], ['03:15', '03:18', '00:02'], ['03:15', '03:18', '00:02']]" 
where 3:15 or 3:18 would be the perfect time to skip toward, if that is how it would work. 

The debug message I get when stuck is:
"bw_plex :: DEBUG :: Checking if .. has a recap with phrase previously on, last season, last episode using subtitles"

Using Ubuntu as a plex server, but if there is no solution for ubuntu, can I disable the subtitle iteration, and only skip trough the theme somehow? It has the length of the theme right I think, and seems to know exactly when to skip to, so I feel like I am close to getting it working.

**Comment by Kapplahmastah** [comment]
Thanks a lot for trying to help me out, and for the example code.
I have no experience with Python at all, only pretty basic knowledge on Java. Am I doing it right at all?


![image](https://user-images.githubusercontent.com/10135312/39676738-e0a3ecac-516f-11e8-96da-19dce8725b03.png)

I tried filling in the right url and token (I think) and the correct showname, season and episode, but it didn't work. I usually run it with username, password and server, (-u -p -s), so I might have messed up the instructions for url and token somehow. I tried changing it to work like that, but didn't work I guess.
